scope3_methodology/publisher.py

```diff
@@ -63,6 +63,12 @@
 )
 parser.add_argument("-v", "--verbose", action="store_true", help="Show derivation of output")
 parser.add_argument("companyFile", nargs=1, help="The company file to parse in YAML format")
+parser.add_argument(
```

Comment:
I think that our default for orgs with no employee or corp emissions data will be a per-request / per-imp number. I wonder if we need the ability to pass either total emissions or emissions per request, meaning this probably needs a couple of flags instead of a second arg at the end.

Reply:
We could definitely do that, and if it's passed in per-request/imp then use that input rather than computing it at the imp/request level. 

It's not clear to me though where we'd get that number from. Do you think that's a default we'll end up generating?

Reply:
yep